types/stylelint/index.d.ts

```diff
@@ -324,6 +324,16 @@ declare module 'stylelint' {
 			 * @internal
 			 */
 			createLinter: (options: LinterOptions) => InternalApi;
+			/**
+			 * Resolves the effective configuation for a given file. Resolves to
+			 * `null` if no config is found.
```

Comment:
[question] Is this a typo?

```suggestion
			 * `undefined` if no config is found.
```

And, just a question, which is better in our public API, `null` or `undefined`?

Reply:
Yes, it's a mistake. I switched it to `undefined` so that it would match the type of `config` in `LinterOptions` but forgot to update it everywhere.

Reply:
I see. Thank you for the explanation!

Reply:
Done in dfbd98e.